<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-PVI9R6Q" failures="1" tests="6" name="testlayer.Alltest" time="4.103" errors="0" timestamp="2023-09-06T17:19:03 GMT-04:00" skipped="1">
  <testcase classname="testlayer.Alltest" name="workhard1" time="0.004"/>
  <system-out/>
  <testcase classname="testlayer.Alltest" name="workhard1" time="0.003"/>
  <system-out/>
  <testcase classname="testlayer.Alltest" name="screenshottest" time="4.073">
    <failure message="expected [hello] but found [Amazon.ca: Low Prices – Fast Shipping – Millions of Items]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [hello] but found [Amazon.ca: Low Prices – Fast Shipping – Millions of Items]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at testlayer.Alltest.screenshottest(Alltest.java:14)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- screenshottest -->
  <system-out/>
  <testcase classname="testlayer.Alltest" name="workhard1" time="0.011"/>
  <system-out/>
  <testcase classname="testlayer.Alltest" name="workhard3" time="0.012"/>
  <system-out/>
  <testcase classname="testlayer.Alltest" name="workhard2">
    <skipped/>
  </testcase> <!-- workhard2 -->
  <system-out/>
</testsuite> <!-- testlayer.Alltest -->
